framework/base/ErrorException.php

```diff
@@ -17,6 +17,8 @@
  */
 class ErrorException extends \ErrorException
 {
+    const E_HHVM_FATAL_ERROR = 16777217;
```

Comment:
need a comment that explains where this comes from. should contain info like given in https://github.com/yiisoft/yii2/pull/8935#discussion_r33483774